### DISCUSSION TITLE
Make sharp optional peer dependency

### DISCUSSION THREAD

**Discussion by navin-moorthy** [vercel/next.js#47727] [Help] [answered]
### Summary

For any deployment with native image loader it is recommeded in the docs to install `sharp` package.

But because of the absence of the optional peer dependency, it was not able to detect the sharp usage linked to next.js in [`knip`](https://github.com/webpro/knip)

In order to detect `sharp` and it's association with `next`, it would be better to mention them in the package.json.

Reference: https://github.com/vercel/next.js/pull/19216 - Any update on this?

**Comment by fikrikarim** [comment, marked as answer]
They added this on version `14.3.0`: https://github.com/vercel/next.js/pull/63321
